scripts/update-cookie-db.cjs

```diff
@@ -81,6 +81,39 @@ const errorHandler = (err) => {
   process.exit(1);
 };
 
+/**
+ * Add keys to the locale file and replace the text with keys in the formattedData.
+ * @param formattedData formatted data
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const addKeysToLocale = async (formattedData) => {
```

Comment:
Is it being used?

Reply:
The callback is not used currently.

Reply:
What is purpose of keeping it?

Reply:
It is kept for future usecase when i18n functions will be implemented, so this callback will help creating keys for the db.

Reply:
Lets add that as a comment.

Reply:
Merging it for now!